src/test/suite/fixtures/recorded/languages/python/chuckEveryArgRed.yml

```diff
@@ -0,0 +1,27 @@
+languageId: javascript
+command:
+  version: 1
+  spokenForm: chuck every arg red
+  action: remove
+  targets:
+    - type: primitive
+      modifier: {type: containingScope, scopeType: argumentOrParameter, includeSiblings: true}
+      mark: {type: decoratedSymbol, symbolColor: default, character: r}
+initialState:
+  documentContents: "\" \".join(word for word in word_list)"
```

Comment:
I guess this one is a good corner case, but would be good to also have a test where there are multiple generator expressions as arguments

Reply:
Added to new commit.